### PR TITLE
Register '-net' application during deployment

### PR DESCRIPTION
**Pull request by Porges**
## Summary of the Pull Request

Adding the `-net` service to app registration so that OneFuzz CLI can access it by changing endpoint.

VS Code reformatted the file a bit.

## PR Checklist
* [ ] Applies to work item: #xxx
* [ ] CLA signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/onefuzz) and sign the CLI.
* [ ] Tests added/passed
* [ ] Requires documentation to be updated
* [ ] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #xxx

## Info on Pull Request

_What does this include?_

## Validation Steps Performed

_How does someone test & validate?_

### CODE REVIEW

File: src/deployment/deploy.py [hunk 1]
```diff
@@ -289,31 +293,25 @@ def create_password(self, object_id: UUID) -> Tuple[str, str]:
...
+    def get_instance_urls(self) -> List[str]:
         # The url to access the instance
         # This also represents the legacy identifier_uris of the application
         # registration
         if self.multi_tenant_domain:
-            return "https://%s/%s" % (
-                self.multi_tenant_domain,
-                self.application_name,
-            )
+            return ["https://%s/%s" % (self.multi_tenant_domain, name) for name in [self.application_name, self.application_name + DOTNET_APPLICATION_SUFFIX]]
```
Comment (chkeita) [inline src/deployment/deploy.py:301]: The "-net" functions are supposed to be temporary until we move everything to dotnet. 
with this changes the "-net" identifierUris and related info will become useless after the transfer.
Can you make this an option specified at deployment.

Comment (Porges) [reply src/deployment/deploy.py:301]: @chkeita I think we should be setting this as long as the `deploy.py` script is deploying the `-net` application separately. Would it be sufficient to add a note next to the application deployment to say to remove this setting once we remove the Python app?

Comment (chkeita) [reply src/deployment/deploy.py:301]: i am thinking about upgrade scenarios for existing deployments. 
If we keep it in the code as-is, we need to add a the code to remove it afterwards when we do the migration. 
We already have a setting in the deployment script that specify which -net function we want to activate. We could make that it dependent on that.

Comment (chkeita) [reply src/deployment/deploy.py:301]: The other options is to make sure we update the identifierUris (and related settings) at each deployment. Currently we only set it on the first time we create the registration.

Comment (Porges) [reply src/deployment/deploy.py:301]: @chkeita at the moment we do perform an update to add new identifier URIs ([see below](https://github.com/microsoft/onefuzz/pull/2090/files#diff-0489104f2cf3e8ff8968514b328b6cb496d9519ee74fa9f24b612ec060f2b4cfR452-R464)) but we never remove any. We could remove them in future?

**Comment by Porges** [comment]
Closing in favour of combining this with #2084.

**Pull request closed without merging**